oem/ibm/pfw-sms/meson.build

```diff
@@ -0,0 +1,12 @@
+libpam = cpp.find_library('pam', required: true)
+try_pfw_sms_deps = [libpam]
+
+executable(
+  'try-pfw-sms-menu',
+  'try_pfw_sms_ibm.c',
+  'pfw_sms_menu_ibm.c',
```

Comment:
Please change the .c extension to .cpp and .hpp. Also the folder structure is under ibm you may omit it from the file name itself.

Reply:
Okay, I had previously understood my code should be in C (not C++).  I will make this change.